deploy/base/deployment/kustomization.yaml

```diff
@@ -13,6 +13,7 @@ configMapGenerator:
     behavior: create
     literals:
       - SWISNAP_ENABLE_APACHE=false
+      - SWISNAP_ENABLE_CRI=false
```

Comment:
should this be disabled?

Reply:
Yes, I think so. In the "deployment" the primary use case is to monitor whole cluster with kubernetes plugin. 
CRI and Docker are by default disabled. Deamonset is kind of deployment meant to monitor containers.